apps/demo/stories/SegmentedControl.stories.tsx

```diff
@@ -1,73 +1,41 @@
 import {DemoConfiguration} from "@config";
-import {Box, SegmentedControl, Text, useStoredState} from "ferns-ui";
+import {Box, SegmentedControl} from "ferns-ui";
 import React, {useState} from "react";
 
-const SegmentedControlDemo = () => {
+export const SegmentedControlDemo = () => {
   return (
     <Box display="flex" width="100%">
-      <SegmentedControl items={["One", "Two"]} selectedItemIndex={0} onChange={() => {}} />
+      <SegmentedControl
+        items={["One", "Two", "Three is a bit longer"]}
+        selectedIndex={0}
+        onChange={() => {}}
```

Comment:
It would be nice if the very first segmented control demo example actually set the selected index and changed the tab unless we have a reason not to?

Reply:
Added!